changelog/unreleased/pull-3436

```diff
@@ -0,0 +1,12 @@
+Enhancement: Improve resilience of local backend to (system) crashes
+
+restic now ensures that files stored using the `local` backend are created
+atomically, that is files are either stored completely or not at all. This
+ensures that no incomplete files are left behind even if restic is terminated
+while writing a file.
```

Comment:
I would say "tries to ensure", because we can't guarantee anything. I don't think this is dramatic at all, LGTM.

Reply:
You're right. Flush is (sadly) only a best effort operation.